packages/discovery-react-components/src/components/DocumentPreview/components/ReactPdfViewer/ReactPdfViewer.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { SFC, useEffect } from 'react';
```

Comment:
Just name this `PdfViewer.tsx` and replace the existing code.

Reply:
Yeah, if we go ahead with `react-pdf`, I'll just replace it, rather than name it `ReactPdfViewer`.